penny_university_frontend/src/components/modal/auth/password.tsx

```diff
@@ -17,7 +17,7 @@ type AuthPasswordModalProps = {
   requestPasswordReset: (payload: {email: string}) => void,
 }
 
-const AuthPasswordModal = ({ email, login }: AuthPasswordModalProps) => {
+export const AuthPasswordModal = ({ email, login, requestPasswordReset }: AuthPasswordModalProps) => {
```

Comment:
I don't think we need this export. In the tests, just import the default like `import AuthPasswordModal from '../../components/modal/auth/password'`. If that doesn't work, then this is fine.

Reply:
That's how it originally was and I had to change it to this. I noticed that `Alerts.test.tsx` was testing the exported `ErrorAlert` rather than the default exported `connect(...)(ErrorAlert)`.